action.yml

```diff
@@ -7,6 +7,9 @@ inputs:
   version:
     description: Version of pnpm to install
     required: false
+  version_file_path:
+    description: "Path to a version file. Eg: '.tool-versions' for asdf or 'package.json' for volta"
+    required: false
```

Comment:
This is problematic. `.tool-versions` and `package.json` are completely different formats and should be treated as different features.

Reply:
Honestly, the only reason I did it this way was to match up with how they do it for node versioning in the referenced lib. I figure if js devs are familiar with any GitHub actions at all, it’s likely that one.

I’ll rethink the inputs throwing out that goal and come back with something else.

Or not. Heh. If I have the time I will.